reference/constraints/Url.rst

```diff
@@ -301,3 +302,76 @@ option to ``true``:
 
 This option uses the :phpfunction:`checkdnsrr` PHP function to check the validity
 of the ``ANY`` DNS record corresponding to the host associated with the given URL.
+
+dnsMessage
+~~~~~~~~~~
+
+.. versionadded:: 2.7
+    The ``dnsMessage`` option was introduced in Symfony 2.7.
+
+**type**: ``string`` **default**: ``The host could not be resolved.``
+
+This message is shown when the ``checkDNS`` option is set to ``true`` and the
+DNS check failed.
+
+.. configuration-block::
+
+    .. code-block:: php-annotations
+
+        // src/AppBundle/Entity/Author.php
+        namespace AppBundle\Entity;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class Author
+        {
+            /**
+             * @Assert\Url(
+             *    dnsMessage = "The host '{{ value }}' could not be resolved."
+             * )
+             */
+             protected $bioUrl;
+        }
+
+    .. code-block:: yaml
+
+        # src/AppBundle/Resources/config/validation.yml
+        AppBundle\Entity\Author:
+            properties:
+                bioUrl:
+                    - Url: { dnsMessage: 'The host "{{ value }}" could not be resolved.' }
+
+    .. code-block:: xml
+
+        <!-- src/AppBundle/Resources/config/validation.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <constraint-mapping xmlns="http://symfony.com/schema/dic/constraint-mapping"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping http://symfony.com/schema/dic/constraint-mapping/constraint-mapping-1.0.xsd">
+
+            <class name="AppBundle\Entity\Author">
+                <property name="bioUrl">
+                    <constraint name="Url">
+                        <option name="dnsMessage">The host "{{ value }}" could not be resolved.</option>
+                    </constraint>
+                </property>
+            </class>
+        </constraint-mapping>
+
+    .. code-block:: php
+
+        // src/AppBundle/Entity/Author.php
+        namespace AppBundle\Entity;
+
+        use Symfony\Component\Validator\Mapping\ClassMetadata;
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class Author
+        {
+            public static function loadValidatorMetadata(ClassMetadata $metadata)
+            {
+                $metadata->addPropertyConstraint('bioUrl', new Assert\Url(array(
+                     'dnsMessage' => 'The host "{{ value }}" could not be resolved.'
```

Comment:
If you want to, you can make that change. Otherwise we will just apply this when merging. :)